### PR TITLE
815 main window resizing

### LINKED ISSUE #815: "Update available" text doesn't wrap properly
When I run master, this is what the "Update available" looks like. I think it should always be on one line and not wrap. Not sure if this is a macOS thing. My guess is that it's not easily fixable. Any ideas, @dcbriccetti ?

*On launch* -  😄
Screen Shot 2019-09-17 at 12 39 08 PM

*Make window bigger* - 😟
Screen Shot 2019-09-17 at 12 39 14 PM

*Make window smaller* -  😭
Screen Shot 2019-09-17 at 12 39 21 PM

### PR DESCRIPTION
**Pull request by dcbriccetti**
Closes #815

Please see the commit comments for details of the changes.

#### What has been done to verify that this works as intended?
Not enough. It looks great on a Mac, but I’ll need help checking on other platforms.
Could someone please check that the underline is gone and that the hyperlink still works when Aggregate or Central is the source or target?

#### Why is this the best possible solution? Were any other approaches considered?
Using a ContentPane and BorderLayout seems best. 

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Should have no effect.

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
No

### CODE REVIEW

**Comment by ggalmazor** [comment]
> Perhaps @ggalmazor can fix the unchecked call to put in makeClickable.

I don't see a way to remove that unchecked call. I think we need to @Supress("unchecked") to the overloaded method. 

Why not assuming that `boolean underline` will always be true and simplify the `if` block? We can extract a method to get the underlined font and isolate the unchecked operation there.

**Comment by dcbriccetti** [comment]
@ggalmazor Thanks for that. What do you think about removing underlines from links throughout Briefcase?

**Comment by ggalmazor** [comment]
> What do you think about removing underlines from links throughout Briefcase?

I don't have a strong opinion about it. I'll let @yanokwa and @kkrawczyk123 fight for it ;)

**Comment by kkrawczyk123** [comment]
I don't have a strong opinion about that too and I am not protesting :)

**Comment by yanokwa** [comment]
If the blue stays, the underline can go.

File: src/org/opendatakit/briefcase/ui/reused/transfer/sourcetarget/target/Central.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void onSelect(Container container) {
   public void decorate(JLabel label) {
     label.setText("" +
         "<html>" +
-        "URL: <a href=\"" + server.getBaseUrl().toString() + "\">" + getDescription() + "</a><br/>" +
+        "URL: <a style='text-decoration: none' href=\"" + server.getBaseUrl().toString() + "\">" + getDescription() + "</a><br/>" +
```
Comment (ggalmazor) [inline src/org/opendatakit/briefcase/ui/reused/transfer/sourcetarget/target/Central.java]: Sorry to be nitpicky, but we should use double quotes for HTML tag attribute values. 

I know it's annoying to escape them :/

Comment (dcbriccetti) [reply src/org/opendatakit/briefcase/ui/reused/transfer/sourcetarget/target/Central.java]: Happy to change, but why do you want those ugly double quotes that use twice as much ink?

**Comment by dcbriccetti** [comment]
Yo, yo, yo, this is ready to go.
(I’m in no rush, but the Github prodder has appeared.)

**Pull request merged**